content/docs/secure/encryption/rekey.md

```diff
@@ -0,0 +1,177 @@
+Labels:          <none>
+Annotations:     pv.kubernetes.io/provisioned-by: sec-isilon.dellemc.com
+...
+Events:                <none>
+```
+
+An associated example Rekey CR for this volume:
+``` 
+apiVersion: "encryption.storage.dell.com/v1alpha1"
+kind: "Rekey"
+metadata:
+  name: "example-rekey"
+spec:
+  persistentVolumeName: "k8s-029a76734f"
+  ```
+
+Apply this CR yaml file to start the rekey process:
+
+```kubectl create -f my-example-rekey.yaml```
+
+### Inspect Status of Rekey
+
+Obtain the status of the rekey:
+
+```kubectl describe rekey my-example-rekey```
+
+```
+Name:         example-rekey
+Namespace:    
+Labels:       <none>
+Annotations:  <none>
+API Version:  encryption.storage.dell.com/v1alpha1
+Kind:         Rekey
+Metadata:
+  Creation Timestamp:  2022-11-11T14:01:51Z
+  Generation:          1
+  Managed Fields:
+    API Version:  encryption.storage.dell.com/v1alpha1
+    Fields Type:  FieldsV1
+    fieldsV1:
+      f:spec:
+        .:
+        f:persistentVolumeName:
+    Manager:      kubectl-create
+    Operation:    Update
+    Time:         2022-11-11T14:01:51Z
+    API Version:  encryption.storage.dell.com/v1alpha1
+    Fields Type:  FieldsV1
+    fieldsV1:
+      f:status:
+        .:
+        f:phase:
+    Manager:         rekey-controller
+    Operation:       Update
+    Subresource:     status
+    Time:            2022-11-11T14:01:51Z
+  Resource Version:  21062146
+  UID:               388a0245-18be-4e35-8618-814f30960b43
+Spec:
+  Persistent Volume Name:  k8s-029a76734f
+Status:
+  Phase:  completed
+Events:   <none>
+```
+If `status.phase` has been set to `completed`, then the rekey was successful.
+
+
+Once the CR has been created, after some time, the status of the rekey can be 
+inspected through the `Status.Phase` field of the rekey custom resource.
+
+The `Status.Phase` field can have the following possible values:
+
+| Status.Phase    | **Description** |
+|-----------------| ----------- | 
+| **initialized** | Preconditions for Rekeying the volume have been met, the volume is attached and encrypted with the CSM encryption driver provisioner.
+| **started**     | The Rekey process has been started with the encryption driver, a connection has been attempted, and a response is expected shortly.
+| **unknown**     | The Rekey process suffered a communication failure while waiting for a response from the encryption driver, and it is possible that the rekey was successfull.
+| **failed**      | The Rekey process has failed, possibly due to a lack of a volume attachment, or no encryption driver could be reached to perform the rekey.
+| **rejected**    | It is not possible to Rekey this volume either because the volume is not encrypted with the specified encryption driver provisioner, or there was no such PV for this volume, or no volume attachments were found.
+| **completed**   | The Rekey successfully completed, which directly implies the KEK of the volume has been changed successfully.

```

Comment:
I suggest removing majority of the details explaining the internals of the process. Also, users don't have to deal with and know about volume attachments to do rekeying. Please change to a more high level description.

Reply:
simplified the text